kqueen_ui/config/utils.py

```diff
@@ -17,34 +16,12 @@ def select_file(config_file=None):
         str: filename to be used as a configuration file
     """
 
-    if not config_file or config_file == 'None':
-        config_file = os.environ.get('KQUEEN_UI_CONFIG_FILE')
```

Comment:
KQUEEN_UI_CONFIG_FILE -> KQUEENUI_CONFIG_FILE

Reply:
take a look on previous code, it was still equal @katyafervent